### PR TITLE
Do not return the sem of periodic geometries when requesting edges

### LINKED ISSUE #140: `volume_factory.loft` stl bug
Running the example I found in the documentation, it seems that the volume_factory.loft function has some kind of problem. The produced stl has a weird thing protruding. I assume it is some kind of intersection happening. This makes problems when trying to process the output with cgal.

example:
image
image

using: 
Splipy 1.4.0
Scipy1.5.4
Numpy 1.19.4

for reference the code I ran
```py
import matplotlib.pyplot as plt
import numpy as np
from splipy import surface_factory, volume_factory
from splipy.io.stl import STL
import os 

srf1 = surface_factory.disc(r=1)
srf2 = 1.5 * srf1 + [0,0,1]
srf3 = 1.0 * srf1 + [0,0,2]
srf4 = 1.3 * srf1 + [0,0,4]
vol = volume_factory.loft(srf1, srf2, srf3, srf4)

# alternatively you can provide all input curves as a list
#all_my_surfaces = [srf1, srf2, srf3, srf4]
#vol = volume_factory.loft(all_my_surfaces)


SAVE = True
if SAVE:
    with STL('test.stl') as myfile:
        myfile.write(vol.swap()) # swap() is to make sure normals are pointing out of the object

os.system('fstl ./test.stl')
```

### PR DESCRIPTION
**Pull request by VikingScientist**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -31,15 +31,8 @@
...
-    ext_modules=cythonize([
-        Extension(
-            'splipy.basis_eval',
-            ['splipy/basis_eval.pyx'],
-            include_dirs=[np.get_include()],
-        )
-    ]),
-    # include_dirs=[np.get_include()],
+    ext_modules=cythonize("splipy/basis_eval.pyx"),
+    include_dirs=[np.get_include()],
```
Comment (TheBB) [inline setup.py]: Was this fixed or not?

File: splipy/volume.py [hunk 2]
```diff
@@ -62,7 +62,12 @@ def faces(self):
         :return: Boundary faces
         :rtype: (Surface)
         """
-        return tuple(self.section(*args) for args in sections(3, 2))
+        boundary_faces = [self.section(*args) for args in sections(3, 2)]
+        for i,b in enumerate(self.bases):
+            if b.periodic > -1:
+                boundary_faces[2*i  ] = None
+                boundary_faces[2*i+1] = None
+        return tuple(boundary_faces)
```
Comment (TheBB) [inline splipy/volume.py:70]: Can we do `tuple(f for f in boundary_faces if f is not None)`, and then ditch the check for `None` in *stl.py*? Or is there some other code that relies on there being 6 faces or something?

File: setup.py (hunk 1 above)
Comment (VikingScientist) [reply setup.py]: No this was a mistake. It should not be included.

File: splipy/volume.py (hunk 2 above)
Comment (VikingScientist) [reply splipy/volume.py:70]: I would expect there to be some end-user code with logic built on this returning 6 faces. It is also very helful to know the ordering (i.e. face[2] would *always* correspond to vmin).

The way I see this is that `SplineModel` can ditch the extra faces (both in a multi- and singlepatch setting), but `Volume` objects should return a predictable amount of entries.

Comment (TheBB) [reply splipy/volume.py:70]: Yeah, that's fine.

File: setup.py (hunk 1 above)
Comment (VikingScientist) [reply setup.py]: Revoked `setup.py` changes

**Pull request merged**